test/input_files/v4.3/passed/passed_body_alt.vcf

```diff
@@ -20,3 +20,7 @@
 1	1800	rs180734498	C	AGT]1:100]	100	PASS	AN=2184;AC=249;AF=0.11	GT:DS:GL	0|0:0.050:-0.13,-0.58,-3.62	0|1:1.000:-2.45,-0.00,-5.00
 1	1900	rs180734498	C	AGT[1:100[	100	PASS	AN=2184;AC=249;AF=0.11	GT:DS:GL	0|0:0.050:-0.13,-0.58,-3.62	0|1:1.000:-2.45,-0.00,-5.00
 1	1900	rs180734498	C	.	100	PASS	AN=2184;AC=249;AF=0.11	GT:DS:GL	0|0:0.050:-0.13,-0.58,-3.62	0|1:1.000:-2.45,-0.00,-5.00
+1	4370	.	G	<*>	.	.	END=4383	GT:DP:GQ:MIN:PL	0/0:25:60:23:0,60,900	0/0:25:45:25:0,42,630
+1	4389	.	T	TC,<*>	213.73	.	.	GT:DP:GQ:PL	0/1:23:99:51,0,36,93,92,86	0/0:24:52:0,52,95,66,95,97
+1	4389	.	C	<*>,<DEL>	.	.	END=4388	GT:DP:GQ	0/0:25:45	0/1:23:99
+1	4390	.	C	*,<*>	.	.	END=4390	GT:DP:GQ	0/0:26:0	0/0:25:60
```

Comment:
Are we really accepting this? Is it correct? 🤔

Reply:
> 5. ALT - alternate base(s): Comma separated list of alternate non-reference alleles. These alleles do not have to be called in any of the samples. Options are base Strings made up of the bases `A,C,G,T,N,*`, (case insensitive) or an angle-bracketed ID String (`“<ID>”`) or a breakend replacement string as described in the section on breakends. The `‘*’` allele is reserved to indicate that the allele is missing due to a an overlapping deletion.